.eslintrc

```diff
@@ -40,7 +40,17 @@
     "no-console": "off",
 
     # Allow console log messages
-    "import/no-extraneous-dependencies": ["error", {"devDependencies": true}]
+    "import/no-extraneous-dependencies": ["error", {"devDependencies": true}],
+
+    # Disable the depricated label-has-for rule
```

Comment:
deprecated :)

Reply:
Even though this is depricated, its still in the AirBnb config, so we need to do some manual workaround temporarily

Reply:
I was talking about the spelling haha -- sorry.

Reply:
🤦‍♂️ Did not catch that